### PR TITLE
Add package and a test

### PR DESCRIPTION
**Pull request by KazuCocoa**
related to https://github.com/appium/python-client/pull/291/files, https://github.com/appium/python-client/issues/295

- Ensured it loads correctly defining `context_test.py`
- Added an `extensions` package introduced in #291 to publish it via pypi

### CODE REVIEW

**Comment by KazuCocoa** [comment]
Missing extention package (0.33)

image

**Comment by KazuCocoa** [comment]
Tests all passed.
A test for a context also worked which failed in 7a0c013 .
I'll merge and release the new version soon.

File: appium/webdriver/webdriver.py [hunk 1]
```diff
@@ -1412,7 +1412,20 @@ def finger_print(self, finger_id):
     # pylint: disable=protected-access
 
     def _addCommands(self):
-        super(WebDriver, self)._addCommands()
+        self.command_executor._commands[Command.CONTEXTS] = \
```
Comment (mykola-mokhnach) [inline appium/webdriver/webdriver.py]: What is the reason to remove these from extensions?

Comment (KazuCocoa) [reply appium/webdriver/webdriver.py]: As below, the missing value happened. https://travis-ci.org/appium/python-client/jobs/469424297

```
self = <selenium.webdriver.remote.remote_connection.RemoteConnection object at 0x7f6dd07009b0>
command = ('getCurrentContext',), params = {'sessionId': '1234567890'}
    def execute(self, command, params):
        """
        Send a command to the remote server.
    
        Any path subtitutions required for the URL mapped to the command should be
        included in the command parameters.
    
        :Args:
         - command - A string specifying the command to execute.
         - params - A dictionary of named parameters to send with the command as
           its JSON payload.
        """
>       command_info = self._commands[command]
E       KeyError: ('getCurrentContext',)
.tox/py37/lib/python3.7/site-packages/selenium/webdriver/remote/remote_connection.py:367: KeyError
```

`('GET', '/session/$sessionId/contexts')` did not stored in `_commands` correctly.
(Thus, I reverted it as a hot fix)

Comment (mykola-mokhnach) [reply appium/webdriver/webdriver.py]: sorry for that. I have created a better fix: https://github.com/appium/python-client/pull/297/files

Comment (mykola-mokhnach) [reply appium/webdriver/webdriver.py]: this multiple inheritance stuff is not so simple though %)

**Pull request merged**